question-service/docker-compose.test.yml

```diff
@@ -37,7 +37,7 @@ services:
             postgres_db:
                 condition: service_healthy
         extra_hosts:
-            - "host.docker.internal:host-gateway"        
+            - 'host.docker.internal:host-gateway'
```

Comment:
possible to explain what's happening here (perhaps by adding a comment)? Curious what's the use of this

Reply:
Actually, I think this is no longer needed, previously the db host is docker.internal and on linux this has to be declared for it to work. Will remove this